### DISCUSSION TITLE
Image zoom in publication page results in blurry zoomed image

### DISCUSSION THREAD

**Discussion by flavio-martinelli** [alshedivat/al-folio#2532] [Q&A] [answered]
I would like to have good resolution zoomable previews of my publications. To do so, I enabled the zoomable option of bib.liquid and modified the preview image size from 200px to 750px (here my bib.liquid Lines 32-41):

```
{% assign entry_path = entry.preview | prepend: '/assets/img/publication_preview/' %}
{%
    include figure.liquid
    loading="eager"
    path=entry_path
    sizes = "750px"
    class="preview z-depth-1 rounded"
    zoomable=true
    alt=entry.preview
 %}
```

However, the zoomed image is much blurrier than even the preview image (as you can see [here](https://flavio-martinelli.github.io/publications/)). How can I fix this?

**Comment by george-gca** [comment, marked as answer]
Related issue #2668.
